modules/core/src/main/scala/org/scalasteward/core/application/Config.scala

```diff
@@ -118,7 +118,7 @@ object Config {
         sandboxCfg = SandboxCfg(
           whitelistedDirectories = args.whitelist,
           readOnlyDirectories = args.readOnly,
-          disableSandbox = args.disableSandbox
+          enableSandbox = args.enableSandbox.getOrElse(args.disableSandbox)
```

Comment:
🤔 

```diff
- enableSandbox = args.enableSandbox.getOrElse(args.disableSandbox)
+ enableSandbox = args.enableSandbox.getOrElse(!args.disableSandbox)
```

Reply:
:facepalm: Of course! Thanks for noticing! If you want to, you can submit a PR changing this. Or I'll do it later.